washboard-ui/playwright.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig, devices } from '@playwright/test';
+
+const PORT = 5173;
+
+export default defineConfig({
+  testDir: './tests/playwright',
+  fullyParallel: true,
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  workers: process.env.CI ? 1 : undefined,
+  reporter: 'html',
```

Comment:
Actually, let's add the  github reporter in CI and switch to the list for local dev.

```suggestion
  reporter: process.env.CI ? [['github'], ['html']] : 'list',
```